src/third_party/progress/progress.hpp

```diff
@@ -208,6 +216,11 @@ class C_Progress_display : public C_Progress
     } // display_tic
 };
 
+inline C_Progress& C_Progress::dummy()
+{
+  static C_Progress null;
```

Comment:
For reference: this is only an issue with MSVC versions < 2015.  https://msdn.microsoft.com/en-au/library/hh567368.aspx#concurrencytable (see Magic Statics)